docs/src/main/sphinx/release/release-397.md

```diff
@@ -0,0 +1,57 @@
+# Release 397 (21 Sep 2022)
+
+## General
+
+* Fix incorrect parsing of invalid values in cast from `varchar` to `timestamp`. ({issue}`14164`)
+* Fix potential incorrect results for queries with a partitioned output which
+  doesn't depend on any column data. ({issue}`14168`)
+* Fix `EXPLAIN (TYPE IO)` query failure for certain queries on empty tables. ({issue}`10398`)
+
+## Security
+
+* Add support for multiple recipients with JSON web token authentication. ({issue}`13442 `)
+* Fix OAuth token refresh causing JSON web token authentication failure. ({issue}`13575`)
+
+## JDBC driver
+
+* Fix potential memory leak when cancelling statements. ({issue}`14176`)
+
+## Delta Lake connector
+
+* Rename the `parquet.experimental-optimized-writer.enabled` configuration
+  property and `experimental_parquet_optimized_writer_enabled` session property
```

Comment:
```suggestion
  property and the `experimental_parquet_optimized_writer_enabled` session property
```

Reply:
This is fine as-is. You don't need to repeat definite articles in lists, e.g. "The cat and the dog are playing" is unnecessarily verbose compared to "the cat and dog are playing."

Reply:
Hmm.. sounds weird to me but oh well